test/multest.jl

```diff
@@ -11,6 +11,7 @@ qv = QuArray(v)
 @assert rawcoeffs(qm'*qm') == m*m
```

Comment:
You are right, although we probably could/should consistently use `coeffs` everywhere (unless we really mean `rawcoeffs`).